drivers/ovsdriver.go

```diff
@@ -432,8 +432,8 @@ func (d *OvsDriver) DeleteMaster(node core.ServiceInfo) error {
 	return nil
 }
 
-// AddBgpNeighbors adds bgp neighbor by named identifier
-func (d *OvsDriver) AddBgpNeighbors(id string) error {
+// AddBgp adds bgp config by named identifier
+func (d *OvsDriver) AddBgp(id string) error {
```

Comment:
I see; then the question really is if someone uses non ovs (i.e. non ofnet), would we need any part of the protocol server outside (not in ofnet), we can take this discussion offline perhaps as we discuss the changes needed for other drivers.

> @jainvipin AddBgp would instantiate a protocol server in ofnet only on vlan mode.So we have to pipe it to the right ofnetAgent for this to happen. In addition to this it will create loopback interface, add bgp endpoints etc.